### PR TITLE
proceed to stage 1

### PR DESCRIPTION
**Pull request by younies**

### CODE REVIEW

**Comment by sffc** [comment]
We should wait for approval from both TG1 (the meeting yesterday) and TG2 (the monthly ECMA-402 meeting).  We have TG1 approval, but not TG2.  So, please don't merge this PR until we have sign-off at next week's TG2 meeting.

**Review by sffc — CHANGES_REQUESTED** [review]
Pending TG2 approval

File: README.md
Comment (sffc) [inline README.md]: Why did this change?

Comment (younies) [reply README.md]: > We should wait for approval from both TG1 (the meeting yesterday) and TG2 (the monthly ECMA-402 meeting). We have TG1 approval, but not TG2. So, please don't merge this PR until we have sign-off at next week's TG2 meeting.

We have received the approval a moment ago :)

Comment (younies) [reply README.md]: Done !

File: index.html
Comment (younies) [inline index.html]: @sffc , shall we change the stage and the date here too?

**Review by sffc — APPROVED** [review]

**Pull request merged**
